crates/core/src/network/net_util.rs

```diff
@@ -45,6 +45,7 @@ pub fn create_tonic_channel_from_advertised_address(address: AdvertisedAddress)
             // todo: Make the channel settings configurable
             Channel::builder(uri)
                 .connect_timeout(Duration::from_secs(5))
+                // todo make http2 keep alive configurable, right now it is using Hyper's defaults
```

Comment:
I think Hyper's default disable the keep alives: https://docs.rs/hyper/latest/hyper/client/conn/http2/struct.Builder.html#method.keep_alive_interval

Reply:
Thanks for the pointer. Will explicitly set the interval to 40s and the timeout to 20s. As a follow-up we'll make this configurable via https://github.com/restatedev/restate/issues/2153.